src/App.js

```diff
@@ -1,25 +1,47 @@
 import React from 'react';
+import { Clock } from './components/Clock';
 
 import './App.scss';
 
-const App = () => {
-  setInterval(() => {
-    const date = new Date();
+class App extends React.Component {
+  state = {
+    isClockVisible: true,
```

Comment:
```suggestion
    isClockVisible: false,
```
зроби щоб спочатку годинник був прихований